pkgs/by-name/si/sispmctl/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, libusb-compat-0_1
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sispmctl";
+  version = "4.11";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    hash = "sha256-dLlKNxAEaxUHDHMR8MrLgVVMhrQidxnMJzPLlscFJXg=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    libusb-compat-0_1
+  ];
+
+  buildInputs = [ ];
+
+  meta = with lib; {
```

Comment:
Please add `mainProgram` even if it's the same as `pname`.

Reply:
done